implementations/rust/ockam/ockam_command/src/node/util.rs

```diff
@@ -250,12 +251,12 @@ pub fn spawn_node(
     Ok(())
 }
 
-/// Retreive the default node name from the global options
-pub fn default_node_name(opts: &CommandGlobalOpts) -> String {
-    opts.state
+/// Retreive the default node name from the CliState::new()
+pub fn default_node_name() -> String {
+    let state = CliState::new().expect("Failed to load CLI state. Try running 'ockam reset'");
+    state
         .nodes
         .default()
-        .unwrap_or_else(|_| panic!("No default Node found"))
-        .config
-        .name
+        .map(|ns| ns.config.name)
+        .unwrap_or_else(|_| "".to_string())
```

Comment:
This doesn't have to change: if there is no default it must panic.

Reply:
Thanks for the review! I tried to panic if there is no default but it seems that the tests are failing because of that. I think this requires more investigation but I suspect that the default config may not be present.

I found that you have implemented a `default_node_name` fn in #4105, hence I updated my code to use your implementation instead. Let me know if it is okay.

Reply:
Oh right, I remember now. It seems to me like a bug from Clap: for some reason, when running any command, the `default_node_name` function is executed (even if the actual command that you are running doesn't use that function).